### ISSUE TITLE
euclidean embedding

### ISSUE THREAD

**Issue by HHalva** [facebookresearch/poincare-embeddings#68]
What is the correct way of creating euclidean embedding with the code? There have been a few questions on this but didnt find an answer, and I would like to make a comparison in my upcoming paper. For now, what I have done is to add Euclidean to the list of embeddings manually (it isnt initially there) -- for now I have not made any changes to the learning rate.

**Comment by lematt1991** [comment]
> For now, what I have done is to add Euclidean to the list of embeddings manually

Yep, this should be enough to train with the euclidean manifold.

> for now I have not made any changes to the learning rate.

You'll probably want to tune the learning rate.

**Comment by HHalva** [comment]
Great thanks!

**Comment by HHalva** [comment]
Doing this, I ran into the issues identified here [https://github.com/facebookresearch/poincare-embeddings/issues/35], would be very helpful to know what's going on
